docs/topics/multiplatform/multiplatform-add-dependencies.md

```diff
@@ -243,6 +243,9 @@ kotlin {
             // SQLDelight will be available only in the iOS source set, but not in Android or common
             implementation("com.squareup.sqldelight:native-driver:%sqlDelightVersion%")
         }
+        wasmJsMain.dependencies {
```

Comment:
How about adding a line of `implementation` with a placeholder, to be consistent with the rest?

Reply:
I think it's enough for a source set placeholder, similar to androidMain above